src/components/samples/ViewSample.tsx

```diff
@@ -721,18 +720,7 @@ const ViewSample = () => {
                   Raw Response
                 </label>
                 <div className="rounded-md border overflow-hidden">
-                  <SyntaxHighlighter
-                    language="json"
-                    style={oneLight}
-                    customStyle={{
-                      margin: 0,
-                      borderRadius: 0,
-                      fontSize: '0.875rem',
-                      lineHeight: '1.5',
-                    }}
-                  >
-                    {sample.raw || ''}
-                  </SyntaxHighlighter>
+                  <CodeBlock language="json" code={sample.raw || ''} />
```

Comment:
Can we showLineNumbers = false for the raw view? I think it makes it harder to read since it's just xml-ish and plain text.